qldbdriver/result_test.go

```diff
@@ -106,6 +140,55 @@ func TestResult(t *testing.T) {
 			})
 		})
 	})
+
+	t.Run("updateMetrics", func(t *testing.T) {
+		t.Run("result does not have metrics and fetch page does not have metrics", func(t *testing.T) {
+			result := Result{consumedIOs: nil, timingInformation: nil}
+			result.updateMetrics(&mockFetchPageResult)
+
+			var mockTimingInformation *TimingInformation = nil
```

Comment:
You can inline this variable for clarity, and it should be zero, not nil.

Reply:
Removed this variable and inlined the zero value.